helpdesk/tests/test_get_email.py

```diff
@@ -20,35 +20,114 @@
     # Python < 3.3
     import mock
 
+unrouted_socks_server = "127.0.0.1"
```

Comment:
I was unsure about this when I started developing this..
Can we be sure the test does not accidentally go awry if somebody has a socks server or email server running on localhost:12345?

However, on travis this will for sure be not the case, so I think as a first approach this is fine.